docker/compose/gen_docker_compose.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python3
+
+# Copyright 2022 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+import shutil
+import sys
+import yaml
+
+from jinja2 import Template
```

Comment:
Note that users may not have `jinja2` installed.  It is not included in the standard python3 install.  Maybe need to add the `python3-jinja2` package to the instructions somewhere.

Reply:
I added a `README.md` with instruction. I'll also add a new wiki page soon. https://github.com/osrf/mbzirc/pull/149/commits/fa653dbb78d88eb551384c196ca30ccc2eb590c9